### PR TITLE
Feature request template added

### LINKED ISSUE #40: [Feature] : Add Issue template for Feature request
@roshan1337d There is a need for feature request template as well

### PR DESCRIPTION
**Pull request by Tushar98644**
fixes #40 

@roshan1337d kindly review

### CODE REVIEW

File: .github/Issue_Template/feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Feature Request 💡
+description: Have any new idea or new feature ? Please suggest!
```
Comment (roshan1337d) [inline .github/Issue_Template/feature_request.yml:2]: Remove the space before `?`

File: .github/Issue_Template/bug.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    attributes: 
+      label: Any additional information? 
+      id: extrainfo
+      description: Any additional information or Is there anything we should know about this bug? 
+    validations: 
+      required: false
+  - type: dropdown
+    id: browsers
+    attributes: 
+      label: What browser are you seeing the problem on?
```
Comment (roshan1337d) [inline .github/Issue_Template/bug.yml:30]: There's an issue with this. Refer to my change requests in your earlier PR for the bug template.

File: .github/Issue_Template/feature_request.yml (hunk 1 above)
Comment (Tushar98644) [reply .github/Issue_Template/feature_request.yml]: done

File: .github/Issue_Template/bug.yml (hunk 2 above)
Comment (Tushar98644) [reply .github/Issue_Template/bug.yml:30]: I have made the changes in other pr

**Comment by Tushar98644** [comment]
@roshan1337d any other changes ?

**Comment by roshan1337d** [comment]
All good 👍

**Comment by Tushar98644** [comment]
> All good 👍

kindly add the labels as well

**Pull request merged**